packages/message-backend/src/create-email/templates/partials/merchantRow.hbs

```diff
@@ -0,0 +1,85 @@
+                                        <tr>
+                                            <td width="100%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{order.merchant.merchantName}}
+                                                                </span>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td width="100%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{order.merchant.merchantStreet}}
+                                                                </span>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td width="100%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{order.merchant.merchantCustomerService}}
+                                                                </span>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td width="100%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{order.merchant.merchantPostalAddress}}
+                                                                </span>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td width="100%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{merchantZip}} {{UPPER order.merchant.merchantName}}
+                                                                </span>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td width="100%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{I18n 'orderConfirmation.merchant.businessId'}} {{order.merchant.merchantBusinessId}}
```

Comment:
We dont have merchantBusinessId info at the moment. This could be added to order data if required by template?

Reply:
Added if statement and created this field to allowed fields.